gin.go

```diff
@@ -267,6 +268,23 @@ func (engine *Engine) LoadHTMLFiles(files ...string) {
 	engine.SetHTMLTemplate(templ)
 }
 
+
+// LoadHTMLFromFS loads HTML files identified by glob pattern
+// from a filesytem interface and associates the result with HTML renderer.
```

Comment:
I can add to the `godoc` a mention of `go:embed` if desired.

The expected usage is to refer to a `go:embed` filesystem, e.g.

```
// embeddedTemplates holds our static web server content.
//go:embed templates/*
var embeddedTemplates embed.FS

// ...

engine.LoadHTMLFromFS(embeddedTemplates, "templates/*.tmpl")
```